src/towncrier/test/test_settings.py

```diff
@@ -153,6 +155,21 @@ def test_towncrier_toml_preferred(self):
         config = load_config(temp)
         self.assertEqual(config.package, "a")
 
+    def test_load_no_config(self):
+        """
+        Check that no exception is raised if no config is found in the base directory.
```

Comment:
Eveything inside the tests is about checking.

The docstring of the test should explain the specific condition and the expected result.

In this way, if this tests fails during a future refactor,
it should be easier to know if there is a bug in the test or a bug in the refactored code.

```suggestion
        Calling the root CLI without an existing configuration file the base directory,
        will exit with code 1 and an info message is sent to stdout.
        No tracebeback is generated.
```

Reply:
Thanks, I updated the docstring.